mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/ProfileFragment.java

```diff
@@ -133,8 +137,17 @@ public void onUserImageEditClicked() {
 
     @OnClick(R.id.btn_profile_bottom_sheet_action)
     public void onEditProfileClicked() {
+        verifyPasscode();
+    }
+
+    private void verifyPasscode() {
         if (getActivity() != null) {
-            getActivity().startActivity(new Intent(getActivity(), EditProfileActivity.class));
+            Intent intent = new Intent(getActivity(), PassCodeActivity.class);
+            intent.putExtra(PassCodeConstants.PASSCODE_INITIAL_LOGIN, true);
+            intent.putExtra(Constants.PASSCODE_NEXT_ACTIVITY,
+                    EditProfileActivity.class.getSimpleName());
+            getActivity().startActivity(intent);
+            Toast.makeText(getActivity(), R.string.edit_credential, Toast.LENGTH_SHORT).show();
```

Comment:
Use our in-house lib `Toaster`

Reply:
@luckyman20 I have updated the PR as per requested.